### PR TITLE
acpica 20150619

### PR DESCRIPTION
**Pull request by Hs-Yeah**
Update acpica to 20150619

### CODE REVIEW

File: Library/Formula/acpica.rb [hunk 1]
```diff
@@ -2,12 +2,12 @@ class Acpica < Formula
...
   homepage "https://www.acpica.org/"
   head "https://github.com/acpica/acpica.git"
-  url "https://acpica.org/sites/acpica/files/acpica-unix2-20150204.tar.gz"
+  url "https://acpica.org/sites/acpica/files/acpica-unix2-20150619.tar_1.gz"
   sha1 "8c5514b1171afb40dca40289581c8ba3f17583e1"
 
   bottle do
     cellar :any
-    sha1 "dbb1b43a1702b3eb2855dd4df1ffbb81a4e5e3c1" => :yosemite
+    sha1 "283c594c4b89db8e3bef8fca25021c52938987ad" => :yosemite
```
Comment (DomT4) [inline Library/Formula/acpica.rb]: Please don't update this. We'll update this line automatically when we pull the latest version in. You may have intended to update the `sha1` beneath the `url` line up top, which does need changing :wink:

Comment (Hs-Yeah) [inline Library/Formula/acpica.rb]: Sorry for my ignorance, corrected.

File: Library/Formula/acpica.rb [hunk 2]
```diff
@@ -2,8 +2,8 @@ class Acpica < Formula
   desc "OS-independent implementation of the ACPI specification"
   homepage "https://www.acpica.org/"
   head "https://github.com/acpica/acpica.git"
-  url "https://acpica.org/sites/acpica/files/acpica-unix2-20150204.tar.gz"
-  sha1 "8c5514b1171afb40dca40289581c8ba3f17583e1"
+  url "https://acpica.org/sites/acpica/files/acpica-unix2-20150619.tar_1.gz"
+  sha1 "283c594c4b89db8e3bef8fca25021c52938987ad"
```
Comment (DomT4) [inline Library/Formula/acpica.rb]: Could you possibly make this `sha256 "12d68f781c9f5c3dfb61ef280f41f0d019d7ba4e914ef916ed3e1d22712ffd52"` instead? Thanks!

**Comment by DomT4** [comment]
The Makefile has changed. Now simply reads:

```make
ifeq ($(OS),efi)
	BUILD_DIRECTORY_PATH = "generate/efi"
	include generate/efi/Makefile.config
	include generate/efi/Makefile.common
else
	BUILD_DIRECTORY_PATH = "generate/unix"
	include generate/unix/Makefile.config
	include generate/unix/Makefile.common
endif
```

So we should just be good to change the current `make` lines to:

```ruby
    ENV.deparallelize
    system "make", "PREFIX=#{prefix}"
    system "make", "install", "PREFIX=#{prefix}"
```

File: Library/Formula/acpica.rb (hunk 1 above)
Comment (DomT4) [inline Library/Formula/acpica.rb]: No need for an apology, It's an easy mistake to make :)

File: Library/Formula/acpica.rb [hunk 3]
```diff
@@ -2,8 +2,8 @@ class Acpica < Formula
   desc "OS-independent implementation of the ACPI specification"
   homepage "https://www.acpica.org/"
   head "https://github.com/acpica/acpica.git"
-  url "https://acpica.org/sites/acpica/files/acpica-unix2-20150204.tar.gz"
-  sha1 "8c5514b1171afb40dca40289581c8ba3f17583e1"
+  url "https://acpica.org/sites/acpica/files/acpica-unix2-20150619.tar_1.gz"
```
Comment (DomT4) [inline Library/Formula/acpica.rb]: I've just noticed our detection code can't automatically handle this tarball version. Can you add a manual version, like this:

```ruby
url "https://acpica.org/sites/acpica/files/acpica-unix2-20150619.tar_1.gz"
version "20150619"
sha256 "12d68f781c9f5c3dfb61ef280f41f0d019d7ba4e914ef916ed3e1d22712ffd52"
```

Thanks!

**Comment by Hs-Yeah** [comment]
Shall we use https://acpica.org/sites/acpica/files/acpica-unix2-20150619.tar.gz for building? But it's corresponding to 20150618.

**Comment by DomT4** [comment]
> Shall we use https://acpica.org/sites/acpica/files/acpica-unix2-20150619.tar.gz for building?

:+1:. It turns out the reason why the other tarball wasn't building is that tar isn't correctly unpacking it, so instead of there being the intended contents inside the unpack it was just `acpica-unix2-20150619.tar_1` in that directory.

> But it's corresponding to 20150618.

Where does it say this?

**Comment by Hs-Yeah** [comment]
After unpacking, we can see the content. It's acpica-unix-20150618.

**Comment by DomT4** [comment]
Heh, so it is :smile_cat:. I think that's just a typo - The `changes` document inside the tarball lists `20150619` and the tarball itself is named `20150619`. Let's drop the version and trust that the tarball name and changes document are correct and the folder name is just a typo.

**Comment by Hs-Yeah** [comment]
I made a diff between the contents of these two files, and found out they are exactly the same. It's a typo indeed.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/3fbd0c6b991a5d80301a45dc2d45458871e31c6a. Thank you for your contribution to Homebrew, and your super-responsiveness in this PR, really appreciate both :star2:.

**Pull request closed without merging**